fenix/app/src/main/res/values/strings.xml

```diff
@@ -1713,6 +1746,8 @@
     <string name="saved_logins_sort_strategy_last_used">Last used</string>
     <!-- Content description (not visible, for screen readers etc.): Sort saved logins dropdown menu chevron icon -->
     <string name="saved_logins_menu_dropdown_chevron_icon_content_description">Sort logins menu</string>
+    <!-- Content description (not visible, for screen readers etc.): Sort saved logins dropdown menu chevron icon -->
```

Comment:
This comment reads as a developer comment: "Sort saved logins dropdown menu chevron icon"
Localizers may not be familiar with technical jargon, so it's helpful to provide clear and straightforward comments that make it easy for them to understand the purpose and function of the string. I would drop that part out of the comment and just leave the first part